UnityClient/Assets/Scripts/PatternCore/Events/GameEventUI.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using UnityEngine;
+
+public class GameEventUI : SingletonDontDestroy<GameEventUI>
+{
+    public static event Action EventUpdateCurrentMiniMap;
+    public static event Action EventUpdateCoordinateMiniMap;
+
+    protected override void Awake()
+    {
+        base.Awake();
+    }
+
+    private void Update()
+    {
+        EventUpdateCurrentMiniMap?.Invoke();
```

Comment:
What is the purpose of this one? We have another one that is triggered by GameManager which invokes the event passing the map name

Reply:
Removed, the event was registered on GameManager.OnMapChanged